### PR TITLE
implement GUI dialog for setting crengine fallback font

### PR DESCRIPTION
**Pull request by frankyifei**

### CODE REVIEW

**Comment by Frenzie** [comment]
I'll have to investigate what makes the automated test fail here. :/

File: frontend/apps/reader/modules/readerfont.lua [hunk 1]
```diff
@@ -209,11 +209,17 @@ end
 
 function ReaderFont:makeDefault(face)
     if face then
-        UIManager:show(ConfirmBox:new{
-            text = T( _("Set default font to %1?"), face),
-            ok_callback = function()
+        UIManager:show(MultiConfirmBox:new{
+            text = T( _([[Set %1 as default or fallback font? The fallback font ]]
```
Comment (Frenzie) [inline frontend/apps/reader/modules/readerfont.lua]: Besides #1617, I only now noticed that this string concatenation doesn't seem to be picked up by Transifex. I'm guessing the problem is [here](https://github.com/koreader/koreader-misc/blob/4bae98523af9820424b0e382a708a9e6d4e5440d/gettext/lua_xgettext.py#L39-L41), but I think simply updating to [the latest from upstream](http://bazaar.launchpad.net/~widelands-dev/widelands/trunk/view/head:/utils/lua_xgettext.py) might solve the problem for us. I might investigate that tonight or this weekend.

Comment (frankyifei) [inline frontend/apps/reader/modules/readerfont.lua]: if that's the problem, is it ok to put a long string here? but that would be ugly and in my understanding not good coding style.

Comment (Frenzie) [inline frontend/apps/reader/modules/readerfont.lua]: In my opinion that'd be neither ugly nor bad style, rather the opposite. But I didn't write the style guide. ;) Anyway, it wasn't an issue [here](https://github.com/koreader/koreader/blob/954c2c941613f41120e8e356aba1f46a9fabbd05/frontend/apps/filemanager/filemanagersetdefaults.lua#L51).

The string concatenation thing just bugs me, even though there's a very simple and obvious fix: don't do it. xD

**Pull request merged**